app/config.py

```diff
@@ -19,6 +19,9 @@
 See here for usage examples:
 https://github.com/google/personfinder/wiki/DeveloperFaq
 """
+import sys
```

Comment:
I'm not sure exactly what this is for (?), but it shouldn't be necessary to modify `sys.path` within our code.

Reply:
@nworden sys.path was present since directory named handlers and utils were having config.py in it and it was supposed to be imported.